src/main/scala/com/metl/model/BackendAdaptor.scala

```diff
@@ -550,3 +551,28 @@ class ResourceCachingAdaptor(sc:ServerConfiguration,cacheConfig:CacheConfig) ext
     super.isReady
   }
 }
+
+class D2LCachingAdaptor(cacheConfig:CacheConfig) {
```

Comment:
Ok created 1m cache over the entire report, but also d2l members as they would otherwise be reloaded for each user in each conv.  Ditching this branch because it's no longer about moving the cache out to be xml configured, but rather just providing multiple caches in the ReportHelper.